test/jdk/TEST.groups

```diff
@@ -323,7 +323,8 @@ jdk_core = \
     :jdk_security \
     :jdk_text \
     :core_tools \
-    :jdk_other
+    :jdk_other \
+    :jdk_core_manual
```

Comment:
Please don't add manual tests to jdk_core.

Reply:
Removed.

Reply:
aren't they already a part of `jdk_core` test group? e.g. `java/net/HugeDataTransferTest.java` is in `:jdk_net` (as `jdk_net` includes `java/net` directory), and `:jdk_core` includes `:jdk_net`?